DTS-STN/Alpha-Site
# Feature/#73 i18n

# Description

[i18n](https://trello.com/c/vKzUIfES/73-73-add-i18n-to-the-alpha-site)

This PR implements i18n for the homepage (index.js) and the Layout component. Language can be toggled using a link on the homepage and all content along with the URL should reflect the active language. This is achieved using [Next Internationalized Routing](https://nextjs.org/docs/advanced-features/i18n-routing) and [next-i18next](https://github.com/isaachinman/next-i18next).

More information in the design doc [here](https://github.com/DTS-STN/Alpha-Site/wiki/Design-Doc-003-Internationalization-Static-Pages)

## Acceptance Criteria

The landing page text has placeholder translations and we can go back and forth between 2 different URL paths to see the different content. 

## Test Instructions

1. `npm run dev` or `npm compose up dev`
2. Navigate to homepage
3. Click language link in header
4. See that content and the URL changes to reflect the currently active locale

## Checklist


## Product and Sprint Backlog

[Trello](https://trello.com/b/ZqWtJSyh/alpha-site-board)
